### DISCUSSION TITLE
default breakpoints

### DISCUSSION THREAD

**Discussion by vanarok** [unocss/unocss#1844] [Q&A] [answered]
Where find default breakpoints for preset-windi?

**Comment by zyyv** [comment, marked as answer]
Hi, you can check this file.  The `preset-wind` theme includes `preset-mini`.
https://github.com/unocss/unocss/blob/98c87b7d2d4e5b13f6a024c4bf43f684c44cd08c/packages/preset-mini/src/_theme/misc.ts#L2-L8
